readme/cla_signatures.json

```diff
@@ -1112,21 +1112,5 @@
       "repoId": 79162682,
       "pullRequestNo": 10093
     },
-    {
-      "name": "danimnunes",
-      "id": 93351930,
-      "comment_id": 1987290500,
-      "created_at": "2024-03-10T16:43:54Z",
-      "repoId": 79162682,
-      "pullRequestNo": 10099
-    },
-    {
-      "name": "7adidaz",
-      "id": 86894852,
-      "comment_id": 1999489659,
-      "created_at": "2024-03-15T11:47:50Z",
-      "repoId": 79162682,
-      "pullRequestNo": 10128
-    }
   ]
-}
\ No newline at end of file
+}
```

Comment:
Please avoid modifying `cla_signatures.json` manually. (See [the instructions above](https://github.com/laurent22/joplin/pull/10136#issuecomment-2001277695) for how to accept the CLA).